resources/js/src/app/components/common/Intersect.vue

```diff
@@ -0,0 +1,86 @@
+<script>
+    const FEATURE_ENABLED = 'IntersectionObserver' in window;
+
+    export default {
+        props:
+        {
+            threshold: {
+                type: Number,
+                default: 0.1
+            },
+            margin: {
+                type: String,
+                default: "0px"
+            }
+        },
+
+        data()
+        {
+            return {
+                isVisible: false,
+                mayObserve: false
+            }
+        },
+
+        created()
+        {
+            if(!FEATURE_ENABLED)
+            {
+                this.isVisible = true;
+                return;
+            }
+
+            this.observer = new IntersectionObserver((entries) => {
+                if(entries[0].intersectionRatio >= this.threshold)
```

Comment:
Ist sichergestellt, dass hier keine minimalen Abweichungen auftreten? Was passiert, wenn man in dem Moment wieder nach oben scroll? ist dann evtl. `intersectionRatio = 0.0999` und `threshold = 0.1`?  Du könntest das noch abfangen, indem du auf `Math.abs(intersectionRatio - threshold) > 0.05` oder so prüfst.